src/components/chatbox/message.module.css

```diff
@@ -42,9 +43,17 @@
 	background: var(--token-color-surface-strong);
 }
 
-.icon {
+.purple {
 	--icon-color: var(--token-color-foreground-highlight-on-surface);
 }
+.strong {
+	--icon-color: var(--token-color-foreground-strong);
+}
+.critical {
+	--icon-color: var(--token-color-foreground-critical);
+
+	color: var(--token-color-foreground-critical);
+}
```

Comment:
It would be better to add support for these color options within `IconTile`, but could be a follow up improvement.

Reply:
Also noting that these styles are supported for [IconTile in HDS](https://helios.hashicorp.design/components/icon-tile) currently.